From: =?utf-8?q?Cristi=C3=A1n_Maureira-Fredes?=
 <cristian.maureira-fredes@qt.io>
Date: Thu, 19 Sep 2024 09:25:05 +0200
Subject: Fix condition to allow 3.13 for wheel creation

Task-number: PYSIDE-2751
Pick-to: 6.8
Change-Id: Iaf6e914bf798371faf5066f4a2d70fe3e2c124e5
Reviewed-by: Christian Tismer <tismer@stackless.com>
(cherry picked from commit a86135803fe51e5702444f8b7dcb2a4dccd3a209)
---
 build_scripts/config.py | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/build_scripts/config.py b/build_scripts/config.py
index 0e7f9fd..27844ee 100644
--- a/build_scripts/config.py
+++ b/build_scripts/config.py
@@ -139,7 +139,7 @@ class Config(object):
         setup_kwargs['zip_safe'] = False
         setup_kwargs['cmdclass'] = cmd_class_dict
         setup_kwargs['version'] = package_version
-        setup_kwargs['python_requires'] = ">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, <3.13"
+        setup_kwargs['python_requires'] = ">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*, !=3.4.*, <3.14"
 
         if quiet:
             # Tells distutils / setuptools to be quiet, and only print warnings or errors.
